vnext/Microsoft.ReactNative/Fabric/Composition/ScrollViewComponentView.cpp

```diff
@@ -919,6 +925,10 @@ void ScrollViewComponentView::OnKeyDown(
 }
 
 bool ScrollViewComponentView::scrollToEnd(bool animate) noexcept {
+  if (!std::static_pointer_cast<const facebook::react::ScrollViewProps>(viewProps())->scrollEnabled) {
+    return true;
```

Comment:
@acoates-ms - returning true for all scroll* scenarios when scrollEnabled = false so the event handlers that triggered these calls consider that the event is handled. Let me know if this approach is wrong.

Reply:
I'd guess that they should be false, so that if you have nested scrollviewers the outer one would scroll based on keyboarding.